### PR TITLE
Restyled 2fa pages

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/javascripts/twofactor.js.coffee [hunk 1]
```diff
@@ -2,3 +2,7 @@
 #= require jquery_ujs
 #= require bootstrap-sprockets
 #= require devise_authy
+
+$(document).ready(() ->
```
Comment (matthewford) [inline app/assets/javascripts/twofactor.js.coffee]: Empty parameter list is forbidden

File: app/views/devise/devise_authy/verify_authy.html.slim [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  br
+
+  = submit_tag I18n.t('submit_token', {:scope => 'devise'}), :class => 'btn btn-primary'
+
+  br
+
+  small
+    = link_to '?', 'https://authy.zendesk.com/hc/en-us/requests/new', :id => 'authy-help', target: "_blank"
+    / br
+    / = authy_request_sms_link
```
Comment (RuslanHamidullin) [inline app/views/devise/devise_authy/verify_authy.html.slim]: Looks like:
```
authy_request_sms_link
```
should be, does it?

File: app/assets/javascripts/twofactor.js.coffee [hunk 3]
```diff
@@ -2,3 +2,7 @@
 #= require jquery_ujs
 #= require bootstrap-sprockets
 #= require devise_authy
+
+$(document).ready( ->
+  $("html").removeClass("no-js").addClass("js")
+)
```
Comment (RuslanHamidullin) [inline app/assets/javascripts/twofactor.js.coffee]: So, our application.js.coffee looks like requiring whole tree
https://github.com/bitzesty/qae/blob/master/app/assets/javascripts/application.js.coffee#L16
```
#= require_tree .
```
not 100% sure, but looks like this one:
```
 $("html").removeClass("no-js").addClass("js")
```
would be called for user forms too. Does it?

File: app/views/devise/devise_authy/verify_authy_installation.html.slim [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  br
+
+  = submit_tag I18n.t('enable_my_account', {:scope => 'devise'}), :class => 'btn btn-primary'
+
+  br
+
+  small
+    = link_to "Help", 'https://authy.zendesk.com/hc/en-us/requests/new', :id => 'authy-help', target: "_blank"
+    / br
+    / = authy_request_sms_link
```
Comment (RuslanHamidullin) [inline app/views/devise/devise_authy/verify_authy_installation.html.slim]: Same note with missing authy_request_sms_link like above ^

File: app/views/devise/devise_authy/verify_authy.html.slim (hunk 2 above)
Comment (hammerfunk) [inline app/views/devise/devise_authy/verify_authy.html.slim]: So I added it back but it just creates a link to a broken page. I don't know if we've added this as a setting. Here's the error:
`No route matches [GET] "/admins/request-sms"`

File: app/assets/javascripts/twofactor.js.coffee (hunk 3 above)
Comment (hammerfunk) [inline app/assets/javascripts/twofactor.js.coffee]: So the 2fa page has it's own layout `twofactor.html.slim` and it uses it's own main JS `twofactor.js.coffee` that only calls a few things, but it doesn't call `#= require_tree .`

File: app/assets/javascripts/application.js.coffee [hunk 5]
```diff
@@ -13,9 +14,11 @@
 #= require browser-check
 #= require vendor/zxcvbn
 #= require vendor/jquery-debounce
-#= require_tree .
+#= require_tree ./frontend
 
 jQuery ->
+  $("html").removeClass("no-js").addClass("js")
```
Comment (RuslanHamidullin) [inline app/assets/javascripts/application.js.coffee]: @hammerfunk just to confirm:
Do we need:
````
 jQuery ->
+  $("html").removeClass("no-js").addClass("js")
```
in app/assets/javascripts/application.js.coffee too?

Comment (hammerfunk) [inline app/assets/javascripts/application.js.coffee]: Yes we do, otherwise it will default to non-js views since all the js only CSS will not activate.

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**